### PR TITLE
New example: Jokes

### PR DESCRIPTION
**Pull request by xyntechx**

### CODE REVIEW

File: examples/jokes/islands/Joke.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const [index, setIndex] = useState(props.index);
+    const btn = tw`px-2 py-1 border(gray-100 1) hover:bg-yellow-100 w-max focus:outline-none mt-4`;
+    const text = tw`p-0 m-2 text-xl`;
+
+    return (
+        <div class={tw`flex flex-col items-center justify-center`}>
+            <p class={text}>{JOKES[index]}</p>
+            <button
+                class={btn}
+                onClick={() => setIndex(Math.floor(Math.random() * 10))}
```
Comment (nikableh) [inline examples/jokes/islands/Joke.tsx]: Change `Math.random() * 10` to `Math.random() * JOKES.length`. See commit 3569e645a5dde7593c9538b346cef13df07593a9

Comment (xyntechx) [reply examples/jokes/islands/Joke.tsx]: Done!

**Comment by xyntechx** [comment]
Any more suggestions?

**Comment by nikableh** [comment]
That's now up to repo maintainers to decide if they want to include such examples of projects done with Fresh.

I think it would be very nice to have several ready-made projects as a reference for _"how it should be done"_.

**Comment by xyntechx** [comment]
> That's now up to repo maintainers to decide if they want to include such examples of projects done with Fresh.
> 
> I think it would be very nice to have several ready-made projects as a reference for _"how it should be done"_.

Yep! Most other frameworks have such examples, which is precisely why I made this PR!

**Pull request closed without merging**